python/py-libusb1/Portfile

```
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               python 1.0
+
+name                    py-libusb1
+version                 3.1.0
+categories              python devel
+platforms               {darwin any}
+supported_archs         noarch
+license                 LGPL-2.1+
+maintainers             nomaintainer
+description             Pure Python wrapper for libusb-1.0.
+long_description        {*}${description}. Supports all transfer types, \
+                        both in synchronous and asynchronous mode.
+
```

`homepage` needs to be set, as per the lint warnings from CI.

```suggestion
homepage                https://github.com/vpelletier/python-libusb1

```

---

A small oversight from the original submitted port; it was using the `github` PG, thanks for catching this :)